tutorials.html

```diff
@@ -0,0 +1,210 @@
+        <strong>Notes:</strong>
+        A two-pronged approach to Spark<br>
+        1.&nbsp;The goal of scientific RDD(sRDD)?&nbsp;The scientific Resilient Distributed Dataset (sRDD), exploits Apache Spark's concept of RDDs for multi-dimensional data representing a scientific measurement that can be subset by time, or by space. The sRDD supports multidimensional data and processing of scientific algorithms in the MapReduce paradigm within a distributed environment.</p>
+        <p>2. sciTensor<br>
+        The sciTensor datatype is a self-documented array that keeps a list of arrays for a variable arrays and maintains associated metadata in a hashmap. The sciTensor is read into the sRDD and the data within is operated on via arithmetic and relational operations. sciTnesor can load data from: HDFS, OpeNDap, and local FS.</p>
+        <p>3. Demo<br>
+        Data visualization:<br>
+        Scala RDD -&gt; Python RDD -&gt; python visualization<br>
+        Use case: Mesoscale convective complexes</p>
+        <ul>
+          <li>Data: brightness temperature data</li>
+          <li>Nodes: areas with a given brightness temperature value and a given size</li>
+          <li>Edges: determined by area overlaps between nodes within consecutive time periods</li>
+          <li>Identify nodes and edges</li>
+          <li>Find cloud elements and connect the cloud elements between frames.</li>
+          <li>Find the subgraphs of cloudy areas that have evolve in time</li>
+        </ul>
+
+        <h3>SciSpark 301: Build your own Climate Metrics</h3>
+
+        <strong>Abstract/Agenda:</strong>
+        <p>We introduce a 3 part course module on&nbsp;<a href="http://sciscpark.jpl.nasa.gov/">SciSpark</a>, our AIST14 funded project for Highly Interactive and Scalable Climate Model Metrics and Analytics. The three part course session introduces a 101, 201, and 301 class for learning how to use Spark for science.</p>
+        <p>SciSpark 301 is a 1.5 hr course in which we will provide lessons learned from our experience in SciSpark as well as a selection of notebooks for attendees to explore, learn from, expand on, and venture out on their own. This session is intended for individuals who have a desire to play with SciSpark and investigate its possible uses in their own work. We plan to have notebooks prepared that show use of a K-means clustering algorithm for identification of Probability Density Functions for climate extremes, the Open Climate WorkBench, and the Climate Model Diagnostic Analyzer. This session will include ample time for more in-depth discussion and problem-solving of attendees’ interests.</p>
+        <ul>
+          <li>P. C. Loikith, J. Kim, H. Lee, B. Linter, C. Mattmann, J. J. D. Neelin, D. E. Waliser, L. Mearns, S. McGinnis. Evaluation of Surface Temperature Probability Distribution Functions in the NARCCAP Hindcast Experiment. Journal of Climate, Vol. 28, No. 3, pp. 978-997, February 2015. doi:10.1175/JCLI-D-13-00457.1.</li>
```

Comment:
Weird - I know I commented on it. Anyway...

I just copied this text wholesale from the ESIP descriptions. Please tell me how you want the references to appear and I'll just do them like that. I have no opinion on this.